CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.2.5.1.0
+- This version of the adapter has been certified with BidMachine 2.5.1.
+
+### 4.2.5.0.0
+- This version of the adapter has been certified with BidMachine 2.5.0.
+
```

Comment:
If we're not releasing 2.5.0, we can probably just remove this entry.

Reply:
we just released it